src/System.Runtime.Loader/ref/System.Runtime.Loader.csproj

```diff
@@ -4,8 +4,10 @@
   <PropertyGroup>
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
     <OutputType>Library</OutputType>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
   <ItemGroup>
+    <Compile Include="System.Reflection.Metadata.cs" />
```

Comment:
I would move the new API into existing single .cs file as @weshaggard suggested.